debian/control

```diff
@@ -0,0 +1,22 @@
+Source: libpam-policycache
+Priority: extra
+Maintainer: Nikki VonHollen <vonhollen@google.com>
+Build-Depends: debhelper (>= 8.0.0), autotools-dev, libglib2.0-dev,
+ libpam0g-dev, automake, libtool, autogen, libscrypt-dev
+Standards-Version: 3.9.2
+Section: libs
+Homepage: https://github.com/google/libpam-policycache
+
+Package: libpam-policycache
+Section: libs
+Architecture: any
+Depends: ${shlibs:Depends}, ${misc:Depends}, libscrypt0
```

Comment:
Nope :(. I'm not sure why. I started playing around with dh_shlibdeps options, but no luck yet.